src/container/srv_target.c

```diff
@@ -157,6 +159,11 @@ cont_child_free_ref(struct daos_llink *llink)
 	D_DEBUG(DF_DSMS, DF_CONT": freeing\n", DP_CONT(NULL, cont->sc_uuid));
 	vos_cont_close(cont->sc_hdl);
 
+	if (cont->sc_agg_ult) {
+		D_ASSERT(d_list_empty(&cont->sc_agg_ult->dtu_list));
+		D_FREE(cont->sc_agg_ult);
```

Comment:
The 'sc_agg_ult' is created when starting aggregation ULT, but freed when evict container cache? What if container aggregation ult start -> stop -> start without evict container cache?